Example code for resilience4j retry

saajn opened the pull request.

thombergs commented:
@saajn I don't understand the reason for the `remoteservice` maven module. I don't see a place where it is called in the other module. Am I missing something?

saajn commented:
> @saajn I don't understand the reason for the `remoteservice` maven module. I don't see a place where it is called in the other module. Am I missing something?

@thombergs I had missed adding the Readme file earlier, added it now. There's one example under "Predicate-based Conditional Retry" which requires a HTTP call. You can look at FlightSearchService.httpSearchFlights() and Examples.predicateExample().

saajn commented:
> > @saajn I don't understand the reason for the `remoteservice` maven module. I don't see a place where it is called in the other module. Am I missing something?
> 
> @thombergs I had missed adding the Readme file earlier, added it now. There's one example under "Predicate-based Conditional Retry" which requires a HTTP call. You can look at FlightSearchService.httpSearchFlights() and Examples.predicateExample().

@thombergs On second thoughts, I realised it's not important to show the http call to convey the idea. I've removed it now..
